libs/ui/src/components/MediaItem/type/VideoMedia.vue

```diff
@@ -34,8 +37,17 @@ const props = withDefaults(
   },
 )
 
+const video = ref()
+
 const autoPlay = computed(() =>
   props.autoplay === undefined ? props.preview : props.autoplay,
 )
 const controls = computed(() => !props.preview)
+
+if (!autoPlay.value && props.src) {
+  useEventListener(video, 'loadeddata', () => {
+    video.value.play()
```

Comment:
Because it will play video in milliseconds to show the thumbnail, the transferred size in the network will be higher than the canary, no?

Reply:
> the transferred size in the network will be higher than the canary, no?

yes, just tested it to make sure 

@preschian do you got any better solution

Reply:
Ideally, capture it on `metadata` workers: https://github.com/kodadot/workers/blob/main/image/src/routes/metadata.ts#L74-L76. And then store it on our indexer

Reply:
like first frame of video? That could be good I guess

Reply:
Sounds like a good Idea!